jwst/ami/utils.py

```diff
@@ -467,23 +356,14 @@ def min_distance_to_edge(img, cntrimg=False):
     img : 2D array
         Input array
 
-    cntrimg : bool
-        If True, only look for the peak pixel near the center of the image
-
     Returns
     -------
     peakx, peaky : integer, integer
         Coordinates of the peak pixel
-
     h : integer
         Distance to the nearest image edge
     """
-    if cntrimg is True:
-        # Only look for the peak pixel at the center of the image
-        ann = makedisk(img.shape[0], 31)  # search radius around array center
```

Comment:
ctrimg=True was unused so I removed it.  It made me uncomfortable that this was hard-coded anyway